plugins/src/webrtcsink/imp.rs

```diff
@@ -1362,48 +1376,49 @@ impl WebRTCSink {
             webrtcbin.set_property("turn-server", turn_server);
         }
 
-        match settings.cc_info.heuristic {
+        let rtpgccbwe = match settings.cc_info.heuristic {
             WebRTCSinkCongestionControl::GoogleCongestionControl => {
-                webrtcbin.connect_closure(
-                    "request-aux-sender",
-                    false,
-                    glib::closure!(@watch element, @strong session_id
-                            => move |_webrtcbin: gst::Element, _transport: gst::Object| {
-
-                        let settings = element.imp().settings.lock().unwrap();
-                        let cc = match gst::ElementFactory::make("rtpgccbwe", None) {
-                            Err(err) => {
-                                glib::g_warning!("webrtcsink",
-                                    "The `rtpgccbwe` element is not available \
-                                    not doing any congestion control: {err:?}"
-                                );
-                                return None;
-                            },
-                            Ok(e) => {
-                                let max_bitrate = (settings.cc_info.max_bitrate as f64 * if settings.do_fec { 1.5 } else { 1.}) as u32;
-                                e.set_properties(&[
+                let rtpgccbwe = match gst::ElementFactory::make("rtpgccbwe", None) {
+                    Err(err) => {
+                        glib::g_warning!(
+                            "webrtcsink",
+                            "The `rtpgccbwe` element is not available \
```

Comment:
Indentation looks fishy, did you run `cargo fmt`?

Reply:
Yes. However, looks like `cargo fmt` don't mind if I manually set second line indentation.
First argument `"webrtcsink"` though is always moved to the new line. Not sure why.

Feel free to reopen the conversation if updated version doesn't look good to you ;)